From: Artemy Kovalyov <artemyko@mellanox.com>
Date: Wed, 31 Aug 2016 05:29:58 +0000
Subject: net/mlx5: Ensure SRQ physical address structure endianness
Patch-mainline: v4.10-rc1
Git-commit: dd257efb1e0f8875ed7e42b88837a8dada0d0e41
References: bsc#1015342 FATE#321688 bsc#1015343 FATE#321689

SRQ physical address structure field should be in big-endian format.

Signed-off-by: Artemy Kovalyov <artemyko@mellanox.com>
Signed-off-by: Leon Romanovsky <leonro@mellanox.com>
Signed-off-by: Leon Romanovsky <leon@kernel.org>
Acked-by: Benjamin Poirier <bpoirier@suse.com>
---
 include/linux/mlx5/srq.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/include/linux/mlx5/srq.h
+++ b/include/linux/mlx5/srq.h
@@ -55,7 +55,7 @@ struct mlx5_srq_attr {
 	u32 lwm;
 	u32 user_index;
 	u64 db_record;
-	u64 *pas;
+	__be64 *pas;
 };
 
 struct mlx5_core_dev;
